website/docs/packages/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Native Packages",
+    "position": 18
+  }
```

Comment:
Couldn't we add the content below as sections in the `package.md` page? The extra sidebar section feels odd to me.